api.py

```diff
@@ -106,18 +106,19 @@ def record_track(self, track_info=None, replace=False):
         uri = track_info['uri']
         duration_ms = track_info['duration_ms']
         duration_s = int(duration_ms / 1000) + 1
-        filename = f"{track_info['name']} - {','.join([artist['name'] for artist in track_info['artists']])}"
+        filename = f"{track_info['name']} - {', '.join([artist['name'] for artist in track_info['artists']])}"
 
-        filename = filename.replace(" ", "_")
```

Comment:
I guess this is more of a personal choice, and I would prefer to keep this pattern for now. Feel free to remove this line on your end if it bothers you, the `spotdl.sh` file in its latest version does not need to be modified to apply this change. Again, this could be an option to add in the future !